app/src/main/java/lt/vilnius/tvarkau/MyProfileActivity.java

```diff
@@ -76,6 +79,11 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
     @Override
     public void onBackPressed() {
+        View view = this.getCurrentFocus();
+        if (view != null) {
+            InputMethodManager imm = (InputMethodManager)getSystemService(Context.INPUT_METHOD_SERVICE);
+            imm.hideSoftInputFromWindow(view.getWindowToken(), 0);
```

Comment:
Possible to put these 2 lines (duplicate in NewProblemActivity) in in a dedicated method and make it reusable.